draft-tuexen-opsawg-pcapng.xml

```diff
@@ -1371,6 +1400,39 @@ Section Header
 
             <t>Example: '0'.</t>
 
+            <t hangText="epb_packetid:"><vspace blankLines="0"/>The
+            epb_packetid option is a 64-bit unsigned integer that uniquely
+            identifies the packet. If the same packet is seen by multiple
+            interfaces and there is a way for the capture application to
+            correlated them, the same epb_packetid value must be used. An
+            example could be where an XDP program on ingress captures a packet,
+            which gets modified and then exits the XDP program. In this case,
+            two packets are in the capture file, which are not identical but
+            the epb_packetid can be used correlate them.</t>
+
+            <t>Example: '0'.</t>
+
+            <t hangText="epb_queue:"><vspace blankLines="0"/>The epb_queue
+            option is a 32-bit unsigned integer that identifies on which queue
+            of the interface the specific packet was received.</t>
+
+            <t>Example: '0'.</t>
+
+            <t hangText="epb_verdict:"><vspace blankLines="0"/>The epb_verdict
+            option stores a verdict of the packet. The verdict indicates what
+            would be done with the packet after processing it. For example, a
+            firewall implemented with eBPF could drop the packet. This verdict
+            can be set by various components, i.e. Hardware, Linux's eBPF TC
+            or XDP framework, etc. etc. The first octet specifies the verdict
+            type, while the following octets contain the actual verdict data,
+            whose size depends on the verdict type, and hence from the value
+            in the first octet. The verdict type can be: Hardware (type
+            octet = 0, size = XXX), Linux_eBPF_TC (type octet = 1, size = 8
```

Comment:
"XXX" presumably meaning "variable".

Reply:
Yes, it means variable, copied this layout from the epb_hash option above. I can change XXX to variable, let me know and I change it for epb_hash also.

Reply:
Yes, the epb_hash option is vague about those hashes in a fashion that doesn't make it clear whether 1) they haven't yet decided how big the hash should be or 2) deliberately left it unspecified to allow different-width hashes to be used.